test/fixtures/tradeQuote.ts

```diff
@@ -153,6 +153,47 @@ export const tradeQuoteFixtures = {
     },
   },
 
+  // Endpoint used by UniswapV3
+  arbitrumBridgeTokenRequest: 'https://bridge.arbitrum.io/token-list-42161.json',
```

Comment:
Source: [Uniswap uses this endpoint][1]

[1]: https://github.com/Uniswap/uniswap-v3-info/blob/2e1313579606aa309e82b681feba005512e1467b/src/constants/lists.ts#L5